Void can't yet compile with edition2021 and there seems to be no reason to
demand it, so partially revert

    https://github.com/PyO3/maturin/commit/4d51112a18a331248bb36c0cb5b4c60c6fe0091a

and make maturin work.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -10,7 +10,7 @@
 license = "MIT OR Apache-2.0"
 keywords = ["python", "cffi", "packaging", "pypi", "pyo3"]
 categories = ["api-bindings", "development-tools::ffi", "command-line-utilities"]
-edition = "2021"
+edition = "2018"
 
 [badges]
 travis-ci = { repository = "PyO3/maturin" }
